pkg/yamlprocessor/yamlprocessor_test.go

```diff
@@ -113,115 +141,14 @@ foo:
 			wantErr: false,
 		},
 	}
-	for _, tc := range testcases {
-		t.Run(tc.name, func(t *testing.T) {
-			got, err := GetValue([]byte(tc.yml), tc.path)
-			assert.Equal(t, tc.wantErr, err != nil)
-			assert.Equal(t, tc.want, got)
-		})
-	}
-}
 
-func TestReplaceValue(t *testing.T) {
```

Comment:
Could you re-add tests for `ReplaceString()`?

Reply:
Oops